docs/root/configuration/other_features/reverse_tunnel.rst

```diff
@@ -223,228 +169,143 @@ For secure reverse tunnel establishment, add a TLS context to the upstream clust
-                -- Priority 3: Extract UUID from Host header (uuid.example.domain)
-                elseif host_header then
-                  local uuid = string.match(host_header, "^([^%.]+)%.example%.domain$")
-                  if uuid then
-                    host_id = uuid
-                    request_handle:logInfo("Extracted UUID from Host header as host_id: " .. host_id)
-                  else
-                    request_handle:logError("Host header format invalid. Expected: uuid.example.domain, got: " .. host_header)
-                    -- Don't set x-computed-host-id, which will cause cluster matching to fail
-                    return
-                  end
-                else
-                  request_handle:logError("No valid headers found: x-node-id, x-cluster-id, or Host")
-                  -- Don't set x-computed-host-id, which will cause cluster matching to fail
-                  return
-                end
-
-                -- Set the computed host ID for the reverse connection cluster
-                headers:add("x-computed-host-id", host_id)
-              end
-        - name: envoy.filters.http.router
-          typed_config:
-            "@type": type.googleapis.com/envoy.extensions.filters.http.router.v3.Router
-
-The egress listener includes a Lua filter as an example of how to implement flexible header-based routing
-to determine which downstream node to route requests to. This is just one of many possible approaches for
-computing the "host_id" from request headers—other options include using different HTTP filters,
-custom formatters, or direct header mapping. The Lua filter example checks multiple headers sequentially
-and sets a computed host ID for the reverse connection cluster, which is then used to look up a socket.
-
-In the example above, the following headers are checked in order of priority:
-
-1. **x-node-id header**: Highest priority—uses the value directly
-2. **x-cluster-id header**: Fallback—used when x-node-id is not present
-3. **Host header**: Second fallback—extracts UUID from format ``uuid.example.domain``
-4. **None found**: Logs error and fails cluster matching
-
-Example Request Flow:
-
-1. **Request with node ID**:
+The example above demonstrates using a :ref:`Lua filter <config_http_filters_lua>` to implement flexible
+header-based routing logic. This is one of several approaches for computing the ``host_id`` from request
+context; alternatives include using other HTTP filters, the ``host_id_format`` field with direct header
+mapping, or custom filter implementations. The Lua filter checks request headers in priority order and
+sets the ``x-computed-host-id`` header, which the reverse connection cluster uses to look up the appropriate
+tunnel connection.
 
-   .. code-block:: http
+The header priority order is:
 
-     GET /downstream_service HTTP/1.1
-     x-node-id: example-node
+1. **x-node-id header**: Highest priority—targets a specific downstream node.
```

Comment:
probably best to use `#.` format so you dont have to renumber on changes - same below